src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/06-daemons.e2e-spec.ts

```diff
@@ -0,0 +1,57 @@
+import { ServicesPageHelper } from '../cluster/services.po';
+
+describe('Daemon list', () => {
+  const services = new ServicesPageHelper();
+
+  beforeEach(() => {
+    cy.login();
+    Cypress.Cookies.preserveOnce('token');
+    services.navigateTo();
+  });
+
+  after(() => {
+    cy.login();
+    Cypress.Cookies.preserveOnce('token');
+    services.navigateTo();
+    services.deleteService('mds.test');
+  });
+
+  it('should stop a daemon', () => {
+    services.navigateTo('create');
+    services.addService('mds');
+
+    services.checkExist('mds.test', true);
+    services.clickServiceTab('mds.test', 'Details');
+    services.checkServiceStatus('mds');
+
+    services.daemonAction('mds', 'stop');
+    services.checkServiceStatus('mds', 'stopped');
+  });
+
+  it('should restart a daemon', () => {
+    services.checkExist('mds.test', true);
+    services.clickServiceTab('mds.test', 'Details');
+    services.daemonAction('mds', 'restart');
+    services.checkServiceStatus('mds', 'running');
+  });
+
+  it('should redeploy a daemon', () => {
+    services.checkExist('mds.test', true);
+    services.clickServiceTab('mds.test', 'Details');
+
+    services.daemonAction('mds', 'stop');
+    services.checkServiceStatus('mds', 'stopped');
+    services.daemonAction('mds', 'redeploy');
+    services.checkServiceStatus('mds', 'running');
+  });
+
+  it('should start a daemon', () => {
+    services.checkExist('mds.test', true);
+    services.clickServiceTab('mds.test', 'Details');
+
+    services.daemonAction('mds', 'stop');
+    services.checkServiceStatus('mds', 'stopped');
+    services.daemonAction('mds', 'start');
+    services.checkServiceStatus('mds', 'running');
+  });
+});
```

Comment:
@pereman2 could you move the tests into `src/pybind/mgr/dashboard/frontend/cypress/integration/workflow/`? There is a `09-services.e2e-spec.ts` which is used to create and delete an mds service. Maybe you can use that suite for testing the daemon actions. Also, the kcli based test suites are present inside that folder so once you add it there these checks will be running in the PR itself (cephadm dashboard e2e tests).

Reply:
@nizamial09 done